travis/setup_xspec_ds9.sh

```diff
@@ -1,32 +1,36 @@
 #!/usr/bin/env bash
 
 # Tarballs to fetch
-DS9_SITE=http://ds9.si.edu/download/centos6/;
-XPA_SITE=http://ds9.si.edu/download/centos6/;
-DS9_TAR=ds9.centos6.7.5.tar.gz;
-XPA_TAR=xpa.centos6.2.1.17.tar.gz;
+ds9_site=http://ds9.si.edu/download/centos6/
+xpa_site=http://ds9.si.edu/download/centos6/
+ds9_tar=ds9.centos6.7.5.tar.gz
+xpa_tar=xpa.centos6.2.1.17.tar.gz
 
 # Fetch them
-wget --quiet $DS9_SITE$DS9_TAR;
-wget --quiet $XPA_SITE$XPA_TAR;
+wget --quiet $ds9_site$ds9_tar
+wget --quiet $xpa_site$xpa_tar
 
 # untar them
-THIS_DIR=$(pwd);
-cd $MINICONDA/bin; tar xf $THIS_DIR/$DS9_TAR; tar xf $THIS_DIR/$XPA_TAR; cd -;
+THIS_DIR=$(pwd)
```

Comment:
Should THIS_DIR be lowercase too? It is not clear to me what rule you are using for deciding on the case of a symbol here ;-)

Reply:
yes :)

In bash I have a tendency to export each and every variable I use, which is not really the way to go, so I am trying to be more disciplined. In this sense, only environment variables are upper case, and you are right, `THIS_DIR` should be lower case.

Reply:
I am reorganizing the script for the inclusion of the xspec and ds9 tests, so I'll take care of that there.